### ISSUE TITLE
Icon request: vero-social

### ISSUE THREAD

**Issue by kevinlanyi** [FortAwesome/Font-Awesome#12758]
Vero is a new social media platform picking up the pace real fast. I'd suggest/request to create a few icons for them, for an easy addition to the current social icons.
Thank you!

https://www.vero.co/press-kit
<img width="135" alt="screen shot 2018-04-02 at 19 08 33" src="https://user-images.githubusercontent.com/34559479/38206115-779433c4-36a9-11e8-8eb3-069cd34c3787.png">
<img width="41" alt="screen shot 2018-04-02 at 19 08 37" src="https://user-images.githubusercontent.com/34559479/38206116-77b25624-36a9-11e8-91ca-fe882781908b.png">

**Comment by Legittti** [comment]
+1

**Comment by Legitti** [comment]
Please do this!

**Comment by Nic787** [comment]
Yes plz!

**Comment by JDSTIGER** [comment]
Almost 2 years & not added yet? 😥

**Comment by Nic787** [comment]
Why is it so long to get this icon?
This should be top priorities... How do I link my Vero account to my website if I don't have the icon?

**Comment by jonathansoltero** [comment]
> Why is it so long to get this icon?
> This should be top priorities... How do I link my Vero account to my website if I don't have the icon?

Might be a money thing, they probably want Vero to pay them to add it.

**Comment by Nic787** [comment]
> > Why is it so long to get this icon?
> > This should be top priorities... How do I link my Vero account to my website if I don't have the icon?
> 
> Might be a money thing, they probably want Vero to pay them to add it.

Did Facebook, Instagram and Twitter paid to get their logo there? I thought FontAwesome was a community project at first. Is that possible for us to create this SVG icon for ourself and integrate it in FA?

**Comment by tagliala** [comment]
> Is that possible for us to create this SVG icon for ourself and integrate it in FA?

Of course. The SVG+JS framework is the easiest way to do that: https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome#svg-framework

---

About other questions: take a look at our [Icon Leaderboard FAQs](https://fontawesome.com/community/leaderboard/new#faqs), they contain a lot of information about how requests work

**Comment by davidkingham** [comment]
How is there still no icon for Vero? The platform is growing like crazy right now...

**Comment by thierrybucco** [comment]
+1111111

**Comment by camaleo** [comment]
I also need this icon!
Vero is becoming more and more used, but all revolutions need people support.
Let people on VERO know about this page, please share this post, it will hopefully help:
https://vero.co/ugrandolini/n1h-xKvVSCV68vqs98nkJv1q

Thank you!

**Comment by qcloud-systems** [comment]
I have a request to add this to our landing pages.  Vero requested.

**Comment by stuttercode** [comment]
I would also like this, but I will not get my hopes up. Requires GitHub community to demand it, not the community that is using the brand. Kind of an odd way of doing things, but I will cast my vote for this regardless.

**Comment by tolgaaaltas** [comment]
Bluesky icon added just in a few months but we're still waiting for Vero. What a shame.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
